subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/LogContent.java

```diff
@@ -39,7 +39,7 @@
     private final static String DOWN_MOVEMENT_WITH_NEW_LINE_PATTERN = "\u001b\\[\\d+B\\n";
     private final static Pattern WORK_IN_PROGRESS_AREA_PATTERN = Pattern.compile(PROGRESS_BAR_PATTERN + "|" + WORK_IN_PROGRESS_PATTERN + "|" + DOWN_MOVEMENT_WITH_NEW_LINE_PATTERN);
     private final static Pattern JAVA_ILLEGAL_ACCESS_WARNING_PATTERN = Pattern.compile("(?ms)WARNING: An illegal reflective access operation has occurred$.+?"
-        + "^WARNING: All illegal access operations will be denied in a future release$");
+        + "^WARNING: All illegal access operations will be denied in a future release\n$");
```

Comment:
Why is this change necessary? Do we need to remove an additional line? If so, we should use `^` to keep it independent of the actual line separator.

Reply:
Yes, the output is `WARNING: xxxx\n` and the original regex can't remove the last newline char.

Reply:
What do you mean using `^` to remove the new line? Please tell me.